docs/spec/staking/state.md

```diff
@@ -2,46 +2,46 @@
 
 ### Pool
 
-The pool is a space for all dynamic global state of the Cosmos Hub.  It tracks
-information about the total amounts of Atoms in all states, moving Atom
-inflation information, etc.
+The pool dynamically tracks information about the total amounts of staking
```

Comment:
What do you mean by "dynamically" - as opposed to "statically"? Further explication requested!

Reply:
updated something close to Jack's suggestion without `dynamically` I think it's just unnecessary. All I meant is that the pool is frequently updated by state transitions.